src/alexandria3k/__main__.py

```diff
@@ -1,536 +1,569 @@
-#!/usr/bin/env python3
```

Comment:
The whole file has had its whitespace changed (I guess line endings) and appears completely changed under `git diff`.  Please adjust Git settings to and the line endings to avoid this.